airgun/entities/activationkey.py

```diff
@@ -86,6 +86,13 @@ def remove_host_collection(self, entity_name, hc_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def enable_repository(self, entity_name, repo_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.repository_sets.click()
+        view.repository_sets.search(repo_name)
+        view.repository_sets.check_box.click()
+        view.repository_sets.actions.fill('Override to Enabled')
```

Comment:
why fill not working for you ? Can you please override that if that is not working for you ?